src/components/DropDown/types.d.ts

```diff
@@ -0,0 +1,3 @@
+export interface IDropDownProps {
+  items: Array<any>;
```

Comment:
can we type it more strict

Reply:
https://github.com/igorizviekov/light-up/pull/7/commits/1a5d2c1645246fff4574c2c630689f33b3316295